confluent_kafka/avro/__init__.py

```diff
@@ -158,3 +178,11 @@ def poll(self, timeout=None):
                     message.offset(),
                     e))
         return message
+
+
+warnings.warn(
+    "The Avro package has been deprecated."
+    " Use the SerializingConsumer and SerializingProducer with"
```

Comment:
it's unfortunate if we need to have new client classes (with more awkward names) to support serializers, i thought we could retrofit them into the old ones?

Reply:
This was at the request of mr @edenhill

Reply:
I'm going to leave the approving to mr @edenhill but want to reiterate that this seems very unfortunate. These are high profile classes and the names are a compromise.

Reply:
agreed

Reply:
`Producer` and `Consumer` are more premium names. it's better to use your premium names for the most high profile uses, and that is here.

Reply:
how is this resolved?

Reply:
We can move this to the base Producer and Consumer classes at a later time.